src/interfaces/libpq/ivytest/oracle_test.sh

```
@@ -0,0 +1,168 @@
+			{
+				dir=/tmp/ivytest_check-$$-$RANDOM
+				(umask 077 && mkdir "$dir")
+			} ||
+			{
+				echo "could not create socket temporary directory in \"/tmp\""
+				exit 1
+			}
+
+			PGHOST=$dir
+			trap 'rm -rf "$PGHOST"' 0
+			trap 'exit 3' 1 2 13 15
```

Shellcheck (0.10.0)

[warning] Trapping signals by number is not well defined. Prefer signal names.

(SC2172)